### PR TITLE
chore: bring latest azure-vnet

### PR DESCRIPTION
**Pull request by rbtr**
**What type of PR is this?**
/kind chore

**What this PR does / why we need it**:
Brings the latest azure-vnet binaries to AB in preparation to release it to AKS

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -300,8 +300,10 @@ unpackAzureCNI() {
 
 #must be both amd64/arm64 images
 VNET_CNI_VERSIONS="
-1.5.11
 1.4.43.1
```
Comment (paulgmiller) [inline vhdbuilder/packer/install-dependencies.sh:303]: 1.4.43.1  and 1.5.11 are what have the (not real) cve in them right? So after our toggle release we can clear these then.

File: vhdbuilder/packer/install-dependencies.sh [hunk 2]
```diff
@@ -315,8 +317,10 @@ done
 #UNITE swift and overlay versions?
```
Comment (paulgmiller) [inline vhdbuilder/packer/install-dependencies.sh:317]: where is dropgz in this file?

**Review by paulgmiller — APPROVED** [review]

File: vhdbuilder/packer/install-dependencies.sh (hunk 1 above)
Comment (rbtr) [reply vhdbuilder/packer/install-dependencies.sh:303]: correct

File: vhdbuilder/packer/install-dependencies.sh (hunk 2 above)
Comment (rbtr) [reply vhdbuilder/packer/install-dependencies.sh:317]: it's in vhdbuilder/packer/components.json

File: vhdbuilder/packer/components.json [hunk 3]
```diff
@@ -363,16 +363,20 @@
       "downloadURL": "https://acs-mirror.azureedge.net/azure-cni/v*/binaries",
       "versions": [
         "1.4.43.1",
```
Comment (ganeshkumarashok) [inline vhdbuilder/packer/components.json:365]: Can we remove 1.4.43.1?

Comment (ganeshkumarashok) [reply vhdbuilder/packer/components.json:365]: Asking to avoid hitting VHD space limits

Comment (rbtr) [reply vhdbuilder/packer/components.json:365]: not yet please, I will remove it when we roll the new one to AKS RP and it stops being used

**Review by ganeshkumarashok — APPROVED** [review]

**Pull request merged**